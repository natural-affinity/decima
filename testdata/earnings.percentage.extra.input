decima -p 12.5 -e 10 80
