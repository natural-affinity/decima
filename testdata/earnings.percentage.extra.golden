You owe: $20.00

